pyi_splash-stubs/__init__.pyi

```diff
@@ -0,0 +1,26 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2022-2022, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+# -----------------------------------------------------------------------------
+
+# https://pyinstaller.org/en/stable/advanced-topics.html#module-pyi_splash
+# https://github.com/pyinstaller/pyinstaller/blob/develop/PyInstaller/fake-modules/pyi_splash.py
+__all__ = ["is_alive", "close", "update_text"]
+
+
+def is_alive() -> bool:
+    ...
+
+
+def update_text(msg: str) -> None:
+    ...
+
+
+def close() -> None:
+    ...
```

Comment:
How come these annotations are in stub/pyi file instead of being inlined?

Reply:
This module only exists at runtime. You can't inline type annotations for the IDE to pickup because `Lib/site-packages/pyi_splash/__init__.py` physically does not exist on disk during development.

If you don't want this I can remove it from here and instead add it to the same external stubs I'd do for PyInstaller.

Reply:
> This module only exists at runtime. You can't inline type annotations for the IDE to pickup because `Lib/site-packages/pyi_splash/__init__.py` physically does not exist on disk during development.

Ah, right, thanks for explanation.

> If you don't want this I can remove it from here and instead add it to the same external stubs I'd do for PyInstaller.

Yes, I would prefer that. Thanks!